bakery/src/pipeline.js

```diff
@@ -10,22 +10,22 @@ module.exports.command = commandUsage
 module.exports.aliases = ['p']
 module.exports.describe = 'builds a pipeline runnable with fly command'
 module.exports.builder = yargs => {	
-  yargs.usage(`Usage: ${process.env.CALLER || 'build.js'} ${commandUsage}`)	
-  yargs.positional('pipelinetype', {	
-    describe: 'type of pipeline',	
-    choices: fs.readdirSync(pipelineDir).map(file => path.basename(file, '.js')),	
-    type: 'string'	
-  }).positional('env', {
-    describe: 'name of environment',
+  yargs.usage(`Usage: ${process.env.CALLER || 'build.js'} ${commandUsage}`)
+  yargs.positional('env', {
+    describe: 'pipeline environment',
```

Comment:
Was the goal of the changes to this file to improve the documentation of the positional arguments on the command, or something functional? (e.g. is it just trying to change the `describe` lines versus the reordering of the args themselves being intentional)

Reply:
reordering the args themselves, if i remember correctly.

Reply:
will confirm - what are your thoughts?

Reply:
I don't think it does reorder them (?) -- at least based on a quick test, but could be user error on my part =). I think to do that you'd need to change line 7 (I'm not super familiar with `yargs`, though). 

However, that being said, I'm not sure the order of `<env> <type>` makes more sense than the current `<type> <env>` (at least in my mind 🤷 )

Reply:
I will double check on this all, thank you!

Reply:
I put things back, like how they are on master now.